cpp/CMakeLists.txt

```diff
@@ -177,8 +177,8 @@ include(cmake/modules/ConfigureCUDA.cmake)
 rapids_cpm_init()
 
 if(NOT BUILD_CPU_ONLY)
-  # thrust before rmm/cuco so we get the right version of thrust/cub
-  include(cmake/thirdparty/get_thrust.cmake)
+  # CCCL before rmm/cuco so we get the right version of CCCL
+  include(cmake/thirdparty/get_cccl.cmake)
```

Comment:
Is there a reason that raft can't use the CCCL in rmm? Can you check this? I'm guessing that the end result is equivalent right now, except that by calling this first we end up bundling CCCL into raft _in addition_ to rmm.

Reply:
This diagnosis is correct, but we discussed offline and decided to live with this to allow easier testing of CCCL. The long-term fix will be to switch to a centralized package for CCCL dependencies that will remove the bundling concern in conda.